kubernetes-model-generator/README.md

```diff
@@ -62,7 +62,7 @@ If everything works well, you would have model upgraded to specified Kubernetes/
  - `cmd/generate/generate.go` contains the main entry point and the descriptor of the schema.
    Add new entries to the `Schema` struct and `schemagen.PackageDescriptor` to generate
    an entry for any additional Go structs.
- - `pkg/schenage` package contains the source files with the logic to generate the schema.
+ - `pkg/schemage` package contains the source files with the logic to generate the schema.
```

Comment:
Shouldn't this be `pkg/schemagen`?

https://github.com/fabric8io/kubernetes-client/tree/master/kubernetes-model-generator/pkg/schemagen